Packs/Slack/Integrations/SlackV3/SlackV3_test.py

```diff
@@ -3291,26 +3289,53 @@ def test_send_message_to_destinations(mocker):
     assert args['body']['text']
 
 
-def test_send_file_to_destinations(mocker):
+def test_send_file_to_destinations_request_args(mocker):
+    """
+    Given:
+        - Sending file to a channel
```

Comment:
```suggestion
        - A file to send
```

Reply:
Improved doc string in commit 7c5b1ee65136fc655e92a3456301506de163b9e6